federation/pkg/federation-controller/ingress/ingress_controller.go

```diff
@@ -168,6 +242,35 @@ func NewIngressController(client federation_release_1_4.Interface) *IngressContr
 			err := client.Extensions().Ingresses(ingress.Namespace).Delete(ingress.Name, &api.DeleteOptions{})
 			return err
 		})
+
+	// Federated configmap updater along with Create/Update/Delete operations.  Only Update should ever be called.
+	ic.federatedConfigMapUpdater = util.NewFederatedUpdater(ic.configMapFederatedInformer,
+		func(client kube_release_1_4.Interface, obj pkg_runtime.Object) error {
+			configMap := obj.(*v1.ConfigMap)
+			configMapName := types.NamespacedName{Name: configMap.Name, Namespace: configMap.Namespace}
+			glog.Errorf("Internal error: Incorrectly attempting to create ConfigMap: %q", configMapName)
+			_, err := client.Core().ConfigMaps(configMap.Namespace).Create(configMap)
```

Comment:
Yes agreed, I originally had it that way, but changed it to this warning to track down a bug.  Will switch it back (in a separate PR).